run.py

```diff
@@ -390,6 +390,14 @@ def conditional_start_servers(self):
             print("Cannot use --client-only and --server-only together.")
             sys.exit(1)
 
+        if use_flask:
+            # Check if flask web client is available
+            try:
+                import neuro_san_web_client  # pylint: disable=unused-import,import-outside-toplevel
+            except ImportError:
+                print("Flask web client is not available. Please install it with `pip install neuro-san-web-client`.")
```

Comment:
We can still use the option to use the Flask web client. This message will tell us to pip install it first.